lib/client/mfa/cli.go

```diff
@@ -35,117 +38,155 @@ import (
 	"github.com/gravitational/teleport/lib/auth/webauthnwin"
 )
 
+// CLIMFAType is the CLI display name for an MFA type.
+type CLIMFAType string
+
+const (
+	CLIMFATypeOTP      = "OTP"
+	CLIMFATypeWebauthn = "WEBAUTHN"
+	CLIMFATypeSSO      = "SSO"
+)
+
 // CLIPrompt is the default CLI mfa prompt implementation.
 type CLIPrompt struct {
```

Comment:
What is the difference between PromptConfig and CLIPrompt? Would it make sense to combine the types? If not, could we document the distinction?

Reply:
PromptConfig contains common configuration shared by by Teleport Connect and other potential prompt implementations, while CLIPrompt also holds config just for CLI - e.g. `Stdin` and `AllowStdinHijack`. Made some changes, let me know if it seems clearer or not.

Reply:
I think we could godoc it a bit more over at PR #47874. The way you put it here is clear, but looking at the types it isn't as much.